### ISSUE TITLE
Select keyboard navigation lack visual feedback

### ISSUE THREAD

**Issue by JesusTheHun** [ant-design/ant-design#26741]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/select/](https://ant.design/components/select/)

### Steps to reproduce
Click on the first select on the page. Once the dropdown appears, press the `up arrow` on your keyboard to select and press enter. As you noticed `Jack` was not highlighted but was selected once enter was hit.

### What is expected?
The option `Jack` is expected to be highlighted when using keyboard navigation.

### What is actually happening?
The navigation happens without visual feedback.

| Environment | Info |
|---|---|
| antd | 4.6.4 |
| React | latest |
| System | MacOS catalina 10.15.6 |
| Browser | all |

**Comment by afc163** [comment]
<img width="153" alt="image" src="https://user-images.githubusercontent.com/507615/93163003-9f449780-f748-11ea-899a-8a16a4b60743.png">

It was highlighted with grey background.

**Comment by JesusTheHun** [comment]
I have 14/10 at both eyes with 120% color depth, and a screen with HDR1000, Adobe 92% and I see nothing, from any angle.

**Comment by afc163** [comment]
You can customize `@select-item-active-bg` to a darker color.

https://ant.design/docs/react/customize-theme

https://github.com/ant-design/ant-design/blob/0de1358924e92a863f70a8b9186203e7ac8c7743/components/style/themes/default.less#L426
